packages/frontend/components/verification/enter-code/component.tsx

```diff
@@ -1,21 +1,24 @@
 import { IconName } from '@enums';
-import { useAppDispatch } from '@hooks';
+import { useAppDispatch, useAuth } from '@hooks';
 import { IconButton } from '@primitives';
 import { PhoneVerification } from 'components/auth/verification/phone-verification';
 import { useTranslation } from 'next-i18next';
 import Image from 'next/image';
 import hand from 'public/images/mocup_hand2.png';
-import { previousVerifyModal } from 'store/verify/actions';
+import { previousVerifyModal } from 'store/modals/actions';
 import { useEffect } from 'react';
 import * as styles from '../styles';
-import { phoneCodeResend } from '../../../store/auth';
+import { getCurrentUser, phoneCodeResend } from '../../../store/auth';
 
 const EnterCodeModal = () => {
   const dispatch = useAppDispatch();
   const { t } = useTranslation();
 
+  const { user } = useAuth();
+
   useEffect(() => {
     dispatch(phoneCodeResend());
+    dispatch(getCurrentUser());
```

Comment:
We can get user data from store, it must be already uploaded. getCurrentUser actions is redundant